helm/values.yaml

```diff
@@ -116,24 +116,22 @@ minio:
             type: AverageValue
   metricsLogger:
     enable: true
-    debug: false
-    image: docker.pkg.github.com/cloudwebmanage/cwm-worker-deployment-minio/metrics_logger
+    image: docker.pkg.github.com/cloudwebmanage/cwm-worker-logger/cwm-worker-logger
     imagePullPolicy: IfNotPresent
     resources:
       requests:
         cpu: "20m"
-        memory: "30Mi"
+        memory: "64Mi"
       limits:
-        memory: "30Mi"
+        memory: "64Mi"
     withRedis: true
-    REDIS_HOST: "localhost"
-    REDIS_PORT: "6379"
-    REDIS_POOL_MAX_CONNECTIONS: "50"
-    REDIS_POOL_TIMEOUT: "5"
-    REDIS_KEY_PREFIX_DEPLOYMENT_LAST_ACTION: "deploymentid:last_action"
-    UPDATE_GRACE_PERIOD_SECONDS: "300"
-    DEPLOYMENT_API_METRICS_FLUSH_INTERVAL_SECONDS: "300"
-    REDIS_KEY_PREFIX_DEPLOYMENT_API_METRIC: "deploymentid:minio-metrics"
+    FLUENTD_CONF: cwm-fluent-default.conf
```

Comment:
Please change this to a meaningful name like:
* `LOG_PROVIDER: ""` (default)
* `LOG_PROVIDER: "elasticsearch"`
* `LOG_PROVIDER: "mongo"`

And add in comments the other two providers, so we can look at the values.yaml and know all the possible options